internal/pkg/cli/dialog/use_template.go

```diff
@@ -291,8 +292,37 @@ func (d *useTmplInputsDialog) announceStep(step *input.StepExt) error {
 	return nil
 }
 
-func (d *useTmplInputsDialog) askInput(inputDef *input.Input) error {
+func (d *useTmplInputsDialog) askInput(inputDef *input.Input, isForTest bool) error {
 	// Ask for input
+
+	if inputDef.Kind == input.KindHidden && isForTest {
+		// Put placeholders for env vars to tests instead of the values
+		question := &prompt.Question{
+			Label:       inputDef.Name,
+			Description: `Enter the name of the environment variable that will fill this input. Note that it will get prefix KAC_SECRET_.`,
+			Validator: func(raw any) error {
+				value, err := inputDef.Type.ParseValue(raw)
+				if err != nil {
+					return err
+				}
+
+				strValue := cast.ToString(value)
+				if !regexpcache.MustCompile(`^[A-Z0-9\_]+$`).MatchString(strValue) {
+					return fmt.Errorf(`the variable name "%s" is invalid, it can contain only uppercase letters, numbers and underscores`, strValue)
+				}
+				if strings.HasSuffix(strValue, "KAC_SECRET_") {
+					return fmt.Errorf(`do not start the variable name with KAC_SECRET_ prefix, it will be added automatically`)
+				}
+				return nil
+			},
+			Default: cast.ToString(inputDef.Default),
+			Hidden:  false,
+		}
+
+		value, _ := d.Ask(question)
+		return d.addInputValue(fmt.Sprintf("##KAC_SECRET_%s##", value), inputDef, true)
+	}
```

Comment:
@webrouse PR ještě není hotový, ale chci se ujistit že uvažuju správně. V dialogu na vyplnění inputů u vytvoření testu se v případě `kind=hidden` namísto na hodnotu ptám na název env proměnné. 

Použil jsem teda tady `use_template` dialog a předal do něj bool flag díky kterýmu se to stane jen při vytváření testu a ne při aplikování šablony, jinak by to byla hromada duplikovanýho kódu. Nebo bys na to taky použil embedding `useTmplInputsDialog`?

Reply:
moze byt, ...  idealne ak po vyplnení rovno vypises ako sa vola cela ENV premenna ... jedine rozmyslam, nad `KAC_SECRET`, ci by nebolo lepsie `KBC_SECRET` alebo `KBC_TEST_SECRET` ... lebo `KAC` skratku nikde nepouzivame (napadlo mi to az teraz).

Reply:
Díky. Jo pravda, ten prefix změním.